index.d.ts

```diff
@@ -1,114 +1,167 @@
 /**
- * main decleration file
+ * main declaration file
  *
- * @author zakaria harti
+ * @autor Star Olaojo
```

Comment:
Let's not override the original author here.

```suggestion
 * @author zakaria harti
```